python/sglang/srt/layers/rotary_embedding.py

```diff
@@ -162,14 +676,13 @@ def __init__(
             / yarn_get_mscale(self.scaling_factor, float(mscale_all_dim))
             * attn_factor
         )
-        self.device = device
         super().__init__(
             head_size, rotary_dim, max_position_embeddings, base, is_neox_style, dtype
         )
 
     def _compute_inv_freq(self, scaling_factor: float) -> torch.Tensor:
         pos_freqs = self.base ** (
-            torch.arange(0, self.rotary_dim, 2, dtype=torch.float, device=self.device)
```

Comment:
You break some changes from this PR https://github.com/sgl-project/sglang/pull/2806

Reply:
cc @zhyncs

Reply:
OK I should make it compatible with CPU device